priv/repo/migrations/20240612081138_add_activity_log_settings.exs

```diff
@@ -0,0 +1,17 @@
+defmodule Trento.Repo.Migrations.AddActivityLogSettings do
+  use Ecto.Migration
+
+  def change do
+    alter table(:settings) do
+      add :activity_log_retention_time, :map
+    end
+  end
+
+  def down do
```

Comment:
I realized that while rolling back, only the `activity_log_retention_time` field was removed, but the record typed `activity_log_settings` remained, meaning that migrating forward again would result in having the activity log settings record with a `null` retention time.

In order to avoid unwanted side effects I thought it is reasonable just deleting the whole record.